src/clean-multihash.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+const bs58 = require('bs58')
+const isIPFS = require('is-ipfs')
+
+module.exports = function (multihash) {
+  if (!isIPFS.multihash(multihash)) {
+    throw new Error('not valid multihash')
+  }
+  if (Buffer.isBuffer(multihash)) {
+    return bs58.encode(multihash)
```

Comment:
We now have the needed methods for this on `js-multihash`, please use those instead of `b58` directly https://github.com/multiformats/js-multihash#api